src/aiperf/controller/system_controller.py

```
@@ -717,24 +748,44 @@ async def _on_process_server_metrics_result_message(
+                    server_metrics_results.endpoints_configured = (
+                        self._server_metrics_endpoints_configured
+                    )
+                    server_metrics_results.endpoints_successful = (
+                        self._server_metrics_endpoints_reachable
+                    )
+                else:
+                    self.debug(
+                        "Received process server metrics result message with no results"
+                    )
+                self._server_metrics_results = server_metrics_results
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)